SU2_CFD/src/output/CNEMOCompOutput.cpp

```diff
@@ -559,7 +559,7 @@ void CNEMOCompOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolv
 
 void CNEMOCompOutput::LoadSurfaceData(CConfig *config, CGeometry *geometry, CSolver **solver, unsigned long iPoint, unsigned short iMarker, unsigned long iVertex){
 
-  if ((config->GetKind_Solver() == NEMO_NAVIER_STOKES)) {
+  if ((config->GetKind_Solver() == NEMO_NAVIER_STOKES && config->GetMarker_All_KindBC(iMarker) != EULER && config->GetMarker_All_KindBC(iMarker) != SYMMETRY_PLANE)) {
     SetVolumeOutputValue("SKIN_FRICTION-X", iPoint, solver[FLOW_SOL]->GetCSkinFriction(iMarker, iVertex, 0));
```

Comment:
Do you only want to include viscous walls? There is a boolean for that in CConfig for each marker.